src/libstd/panic.rs

```diff
@@ -21,6 +21,8 @@ use sync::{Arc, Mutex, RwLock};
 use sys_common::unwind;
 use thread::Result;
 
+pub use panicking::{take_handler, set_handler, PanicInfo, Location};
```

Comment:
Would it be possible to define these in this module? They seems closely related to it at least.